starters/adapters/azure-swa/adapters/azure-swa/vite.config.ts

```diff
@@ -16,8 +16,7 @@ export default extendConfig(baseConfig, () => {
       },
     },
     ssr: {
-      target: 'webworker',
-      noExternal: true,
+      noExternal: /.*/,
```

Comment:
why /.*./? we dont do this in other node deployments

Reply:
The other option would be to generate a `azure-functions/render/package.json` that includes all necessary runtime dependencies and `npm install` those. I've tried this as well but this results in bigger bundles. Bigger bundles mean slower start times for the Azure Functions. 

The config ` ssr: { noExternal: /.*/ }` inlines all third-party dependencies but still uses the Node.js provided modules. This works quite well for my apps so far. See https://github.com/derkoe/qwik-todos/ for an example.

Reply:
This also fixes the e2e tests https://github.com/BuilderIO/qwik-city-e2e/pull/6

Reply:
Ok! let's do it!

Reply:
Thanks for the explanation and all the great work with Azure!!